behat/features/ci_build.feature

```diff
@@ -14,3 +14,16 @@ Feature: CI Build
     | Gizra/Gizra             |
     | drupal/drupal           |
     | DavidKohav/test-example |
+
+  @api
+  Scenario: Check after disabling of the CI build a new CI build item is removed.
+    Given I login with user "admin"
```

Comment:
tests shouldn't not be with admin - can we create a new group and do each test on different group?